lips/lip-7.md

```diff
@@ -0,0 +1,53 @@
+
+## Formation of a Technical Working Group
+Technical Working Groups (TWG) are formed to address or explore specific technical topics or issues as they relate to Libra. TWGs collect information and expert input on a particular topic, discuss material changes to the Libra Payment Network (LPN), and create outputs in the form of recommendations, reports, and Libra Improvement Proposals (LIPs).
+
+The proposal to form a technical working group can be made by any Association member. The proposal must include, but is not limited to:
+The group’s scope and goal
+The limits of its focus
+The expected final output, typically as a LIP, report, or recommendation
+Expected duration of operation
+The initial facilitator (up to two individuals)
+A brief description of ideal desired member profiles so other Association members can join
+
+Once the proposal is created, the facilitator will circulate it in public slack channels relevant to the topic. The facilitator is also encouraged to proactively share the proposal with Association members that are likely to be impacted by this TWG or that could contribute to it. The facilitator must circulate the proposal over 1-2 weeks for sufficient interest to gather.
+
+After gathering interest, the facilitator must choose members to be part of the TWG, add them to the proposal, and identify their roles (co-facilitator, contributor, reviewer, etc.). The proposal must have representation from a minimum of 3 Association members to build collaboration and as a tool to determine relevance to the entire association, rather than an individual member.
+
+The facilitator is responsible for promoting and explaining the proposed TWG to Libra members and must lead it through the voting process by the Technical Steering Committee (TSC). They are also responsible for recruiting other members to the TWG as needed on an ongoing basis, and must help find a replacement facilitator if they must cease work on the initiative.
+
+Once the TWG proposal is presented to the TSC, a simple majority vote in favor forms the TWG. The TWG can then proceed in its stated work and make decisions as needed in the areas covered by its focus.
+
+## Decision Making in a TWG
+The TWG facilitator owns the agenda, meetings, and organization of the TWG, but is not a final decision maker. They are there to drive and facilitate discussion, and help allocate resources so that recommendations, reports, or draft LIPs can be created. They are also responsible for providing a bi-weekly update to the association on the progress the TWG is making and any associated documentation being produced. Bi-weekly updates should be posted in the Libra portal and distributed via email/slack to all Association members.
+
+If there are material changes to the TWG’s scope, timeline, or expected output, a revised proposal must be submitted to the TSC for re-approval.
```

Comment:
The timeline one is a bit strange to me.  If a working group isn't making progress quickly enough, now they need to be re-approved by the TSC to continue their work?

Reply:
Fair point. I also removed expected output because that can also change over time. Really the only thing we need to watch out for is material changes in scope because we shouldn't have a TWG get approval to work on one thing and then pivot to something completely different without getting approval for the new direction.